apps/snitch_core/priv/repo/seed/variation_theme.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Snitch.Seed.VariationTheme do
+
+  alias Snitch.Core.Tools.MultiTenancy.Repo
+  alias Snitch.Data.Schema.VariationTheme
+  alias Snitch.Data.Schema.OptionType
+
+  defp get_option_type do
+    Repo.all(OptionType)
+  end
+
+  def seed!() do
+    option_types = get_option_type
+    option_ids = for option <- option_types, do: to_string(option.id)
+    create_variation_theme("color", [Enum.at(option_ids, 0)])
+    create_variation_theme("size", [Enum.at(option_ids, 1)])
```

Comment:
@jyotigautam `Enum.at` is wrong. You can never be sure of using indexes to find the values.